docker/depends/pecan.depends

```diff
@@ -25,9 +25,9 @@ install2.r -e -s -n -1\
     bit64 \
     car \
     coda \
-    data.table \
     dataone \
     datapack \
+    data.table \
```

Comment:
This is strange -- Did you change this by hand or did running `./scripts/generate_dependencies.R` change the sort order? I wonder if this is secretly locale-dependent...

Reply:
I didn't change this by hand..should I change it back by hand tho?

Reply:
I guess so. I've opened #2515 to track this for later fixing -- would you be willing to report your sessionInfo() there?